class.jetpack.php

```diff
@@ -1515,6 +1515,9 @@ public static function get_active_plan() {
 		// get available features
 		foreach ( self::get_available_modules() as $module_slug ) {
 			$module = self::get_module( $module_slug );
+			if ( ! isset( $module ) || ! is_array( $module ) {
```

Comment:
Missing parens closing the whole if condition here

Reply:
🤦‍♂️ Thank you!